### PR TITLE
Feature/add pancake

### PR DESCRIPTION
**Pull request by enrinal**

### CODE REVIEW

File: config.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+port:8080
```
Comment (enrinal) [inline config.sh:1]: port change to 8088, please

File: model/pancake.go [hunk 2]
```diff
@@ -0,0 +1,5 @@
+type Pancake struct {
+	ID int
+	Name string
+	Size int
```
Comment (enrinal) [inline model/pancake.go:4]: Size in string not int

File: model/pancake.go [hunk 3]
```diff
@@ -0,0 +1,5 @@
+type Pancake struct {
+	ID int
```
Comment (Kadigas) [inline model/pancake.go]: ID must be a string

**Review by HqAlfath24 — APPROVED** [review]

**Review by Kadigas — CHANGES_REQUESTED** [review]

File: model/pancake.go (hunk 3 above)
Comment (enrinal) [reply model/pancake.go]: okaay will be fix in next commit

Comment (enrinal) [reply model/pancake.go]: @Kadigas please review again

**Review by Kadigas — APPROVED** [review]

**Pull request merged**